docs/Using Randomness/vrf-contracts.md

```diff
@@ -99,3 +101,37 @@ Fee parameters are configured in the coordinator contract and specify the premiu
 |Minimum Confirmations|3|
 |Maximum Confirmations|200|
 |Maximum Random Values|500|
+
+### Polygon (Matic) Mainnet
+
+Polygon is not yet available in the Subscription Manager. Use [subscription management functions](/docs/chainlink-vrf/example-contracts/#modifying-subscriptions-and-configurations) in your smart contract to create and modify subscriptions on Polygon.
+
+|Item|Value|
+|---|---|
+|LINK Token|[`0xb0897686c545045aFc77CF20eC7A532E3120E0F1`](https://polygonscan.com/address/0xb0897686c545045aFc77CF20eC7A532E3120E0F1)|
+|VRF Coordinator|[`0xae975071be8f8ee67addbc1a82488f1c24858067`](https://polygonscan.com/address/0xae975071be8f8ee67addbc1a82488f1c24858067)|
```

Comment:
I think we should use the EIP-55 checksummed address of `0xAE975071Be8F8eE67addBC1A82488F1C24858067` because it significantly reduces the chance of typos significantly.

Reply:
Done